lib/text/ui_text_displayer.js

```diff
@@ -46,6 +46,18 @@ shaka.text.UITextDisplayer = class {
     /** @type {HTMLElement} */
     this.textContainer_ = shaka.util.Dom.createHTMLElement('div');
     this.textContainer_.classList.add('shaka-text-container');
+
+    /* Set the subtitles text-centered by default. */
```

Comment:
I like the contents of the comments, but our convention is to use single-line-style comments (e.g. `//`) whenever possible.

Reply:
I'm going to go change it!